pandas/core/generic.py

```diff
@@ -2256,6 +2256,7 @@ def to_excel(
         ExcelWriter : Class for writing DataFrame objects into excel sheets.
         read_excel : Read an Excel file into a pandas DataFrame.
         read_csv : Read a comma-separated values (csv) file into DataFrame.
+        io.formats.style.Styler.to_excel : Add styles to Excel sheet.
```

Comment:
Can we link to Styler.to_excel instead of writing down the whole path?

Reply:
I followed what was mentioned in [#48089](https://github.com/pandas-dev/pandas/issues/48089#issuecomment-1216027713).

Reply:
Ah got you, thx. What I meant is:

:meth:`~io.formats.style.Styler.to_excel`, which would remove the io.. path

Reply:
You can ignore my comment, was not aware that this was done automatically